lib/AST/NodeFactory/NodeFactory.cpp

```diff
@@ -26,12 +26,6 @@ THE SOFTWARE.
 #include "stp/AST/ASTKind.h"
 #include "stp/STPManager/STPManager.h"
 
-stp::ASTNode NodeFactory::CreateTerm(stp::Kind kind, unsigned int width,
```

Comment:
Why was this removed? Is it unused?

Reply:
I said in the pull request "Remove unused NodeFactory::CreateTerm that accidentally called itself."

The only use of the function was to recursively call itself with exactly the same args! Presumably this was missed in a cleanup somewhere.